lib/semian/atomic_integer.rb

```diff
@@ -0,0 +1,28 @@
+module Semian
+  class AtomicInteger #:nodoc:
+    attr_accessor :value
+
+    def initialize(_name, _permissions)
```

Comment:
Are you suggesting just making it a 0 parameter initializer? I'd like to keep the interface the same though.